projects/templates/src/lib/components/po-page-dynamic-table/interfaces/po-page-dynamic-table-actions.interface.ts

```diff
@@ -163,4 +164,21 @@ export interface PoPageDynamicTableActions {
    *
    */
   beforeDetail?: string | ((id?: string, resource?: any) => PoPageDynamicTableBeforeDetail);
+
+  /**
+   * @description
+   *
+   * Rota ou método que será chamado antes de executar o evento de exclusão em lote (removeAll).
+   *
+   * Tanto o método como a API receberão uma lista com as chaves dos recursos que serão excluídos
+   * e devem retornar um objeto com a definição de `PoPageDynamicTableBeforeRemoveAll`.
```

Comment:
- Seria interessante colocar na documentação que o recurso retornado do `beforeRemoveAll` será o parâmetro que vai ser repassado para o `removeAll`, para o usuário saber qual dos dois vai ser aceito para a remoção na tabela. O que você acha da sugestão abaixo?
```suggestion
   * Método/URL que deve ser chamado antes de executar o evento de exclusão em lote (removeAll).
   *
   * Tanto o método como a API receberão uma lista com as keys dos recursos que serão repassados para o 
   * `removeAll` para serem excluídos e devem retornar um objeto com a definição de 
   * `PoPageDynamicTableBeforeRemoveAll`.
```
- Também não é importante destacar que é uma Api/Url/Endpoint? Também pode ser usado o termo rota para esse caso?